bucket/schemacrawler.json

```diff
@@ -0,0 +1,27 @@
+{
+  "version": "16.16.5",
+  "description": "Free database schema discovery and comprehension tool",
+  "homepage": "https://www.schemacrawler.com/",
+  "license": "EPL-2.0",
+  "architecture": {
+    "64bit": {
+      "url": "https://github.com/schemacrawler/SchemaCrawler-Installers/releases/download/v16.16.5/SchemaCrawler-16.16.5.msi",
+      "hash": "89726fd5adbcb31cb0476873318e90f09b081c2ecd998a19ec6b0d197266aac4"
+    }
+  },
+  "bin": "SchemaCrawler\\SchemaCrawler.exe",
```

Comment:
```suggestion
  "extract_dir": "SchemaCrawler",
  "bin": "SchemaCrawler.exe",
```

Reply:
That will not work, @rashil2000 - an extra SchemaCrawler directory is created by extracting the MSI.

Reply:
Sorry - you are right. I am going to make that change. Do I need to make it in the updates section as well?

Reply:
Yes, this is for that directory itself. Try it.